### PR TITLE
Master admob

### PR DESCRIPTION
**Pull request by cerberillo**
Changes
* Added Admob adapter
* Changed startTracking method to accept only `ViewGroup` instead of `View`

### CODE REVIEW

**Comment by RagnarR** [comment]
👍

File: mediation.demo/src/main/java/net/pubnative/mediation/adapter/network/AdMobNetworkAdapter.java [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+
+    //==============================================================================================
+    // AdMobNetworkAdapter methods
+    //==============================================================================================
+    protected void createRequest(Context context, String unitId) {
+
+        Log.v(TAG, "createRequest");
+        AdLoader adLoader = new AdLoader.Builder(context, unitId)
+                .forAppInstallAd(this)
+                .withAdListener(new NativeAdListener())
```
Comment (anshumansworld) [inline mediation.demo/src/main/java/net/pubnative/mediation/adapter/network/AdMobNetworkAdapter.java]: `new NativeAdListener()` is little bit confusing. We don't want to capture events?

Comment (anshumansworld) [inline mediation.demo/src/main/java/net/pubnative/mediation/adapter/network/AdMobNetworkAdapter.java]: Just saw that below, leave it :)

File: mediation.demo/src/main/java/net/pubnative/mediation/adapter/network/AdMobNetworkAdapter.java [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+    //----------------------------------------------------------------------------------------------
+
+    @Override
+    public void onAppInstallAdLoaded(NativeAppInstallAd nativeAppInstallAd) {
+
+        Log.v(TAG, "onAppInstallAdLoaded");
+        AdMobNativeAppInstallAdModel wrapper = new AdMobNativeAppInstallAdModel(nativeAppInstallAd);
+        invokeLoaded(wrapper);
+    }
+}
```
Comment (anshumansworld) [inline mediation.demo/src/main/java/net/pubnative/mediation/adapter/network/AdMobNetworkAdapter.java]: Formatting of class is not correct. Alignment of code :)

Comment (RagnarR) [inline mediation.demo/src/main/java/net/pubnative/mediation/adapter/network/AdMobNetworkAdapter.java]: hmm, but for me looks good. What exact you don't like?

Comment (anshumansworld) [inline mediation.demo/src/main/java/net/pubnative/mediation/adapter/network/AdMobNetworkAdapter.java]: yes, it's correct. I don't know what I checked :D 
anyway removing this comment.

**Comment by anshumansworld** [comment]
👍

**Comment by bullshit74** [comment]
u all can kiss my ass....

**Comment by RagnarR** [comment]
👍

**Comment by cerberillo** [comment]
This got reviewed and approved, merging

**Pull request merged**